optuna/storages/_rdb/storage.py

```diff
@@ -814,6 +816,22 @@ def _lift_numerical_limit(value: Optional[float]) -> float:
             return float(np.sign(value) * float("inf"))
         return value
 
+    @staticmethod
+    def _convert_intermediate_value_representation(
+        intermediate_value_model: models.TrialIntermediateValueModel,
+    ) -> float:
```

Comment:
The name of this function might be a bit ambiguous since it does not indicate how this function convert the representation. I don't have any particular ideas, but we may be able to name after keywords like float type, such as `_revert_float_value_of_intermediate_value`. Or, `_revert_intermediate_value` for simplicity.

Reply:
Yes, that makes sense. Especially `_revert_float_value_of_intermediate_value` sounds better. Or we may use:

* `_construct_float_from_intermediate_values_model` 
* `_to_intermediate_value_from_sqlalchemy_model`

Reply:
Yes, that makes sense. Especially `_revert_float_value_of_intermediate_value` sounds better. Or we may use:

* `_construct_float_from_intermediate_values_model` 
* `_to_intermediate_value_from_sqlalchemy_model`

Reply:
Thank you for suggesting the candidates.

> `_to_intermediate_value_from_sqlalchemy_model`

This one is easier to understand  for me, but all of them are descriptive. I guess this method is private and we can rename it if necessary, so please select one of them.

Reply:
Thanks! I renamed it to `_to_intermediate_value_from_sqlalchemy_model` at bc1af1bc8.